src/pages/docs/v2/08-SDK/08-other-exports.md

```diff
@@ -26,11 +26,8 @@ A union type comprised of all types that can be cast to a JSBI instance.
 ```typescript
 import { ChainId } from '@uniswap/sdk'
```

Comment:
Does this need to be updated to talk about a `@vexchange/sdk` package? @KennethAshley

Reply:
Yep. Ive created our own package. Its “vexchange-sdk”.

Reply:
I'll update